runtime/astar/src/lib.rs

```diff
@@ -263,6 +265,68 @@ impl pallet_custom_signatures::Config for Runtime {
     type UnsignedPriority = EcdsaUnsignedPriority;
 }
 
+parameter_types! {
+    pub const BlockPerEra: BlockNumber = 1 * DAYS;
+    pub const RegisterDeposit: Balance = 100 * ASTR;
+    pub const DeveloperRewardPercentage: Perbill = Perbill::from_percent(1);
+    pub const MaxNumberOfStakersPerContract: u32 = 512;
+    pub const MinimumStakingAmount: Balance = 50 * ASTR;
```

Comment:
I think we said 500 ASTR. Since 5DOT was the min for crowdloan and token distribution was ~100 per DOT

Reply:
Yes correct! 500 ASTR

Reply:
Fixed